app/routes.py

```diff
@@ -0,0 +1,286 @@
+from flask import abort,Blueprint,jsonify,request,make_response
+from app.models.customer import Customer
+from app.models.video import Video
+from app.models.rental import Rental
+from app import db
+import requests, os
+from dotenv import load_dotenv
+from datetime import datetime, timezone, timedelta, date
+
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+videos_bp = Blueprint("videos", __name__, url_prefix="/videos")
+rentals_bp = Blueprint("rentals", __name__, url_prefix="/rentals")
+load_dotenv()
+
+
+@customers_bp.route("", methods=["GET", "POST"])
+def handle_customers():
+
+    if request.method == "GET":
+        customers = Customer.query.all()
+        
+        customers_response = []
+
+        for customer in customers:
+            customers_response.append(customer.to_dict())      
```

Comment:
This works well and is very readable, but if you'd like to start incorporating list comprehensions into your code, a loop like this that builds a list is a great place to start:
```suggestion
        customers_response = [customer.to_dict() for customer in customers]
```